src/Reporter/Reporter.php

```diff
@@ -35,21 +35,21 @@ public function __construct()
 
     public function progress($current, $total)
     {
-        echo sprintf("Reviewing file %d of %d.\r", $current, $total);
+        echo sprintf("Reviewing %d of %d." . PHP_EOL, $current, $total);
```

Comment:
> It doesn't clear the line, except on Windows.

Nope, but it does return the cursor to the beginning of the line before the next write. If I'm not mistaken that's how in the demo it's updating the review count without printing a new line for each one.